ProcessMaker/processmaker
# QA Observation Bugfix/FOUR-12964: There is no option to clear the line input of search in Process Browser

## How to Test
-  Have a process
1. Go to Launch Pad
2. Click on category with processes
3. Check the search engine
4. Check the "X" icon to clear the input search

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-12964

## Code Review Checklist
ci:deploy
ci:next
